tests/integration/tier0/single-yum-transaction-validation/test_single_yum_transaction_validation.py

```diff
@@ -75,7 +78,13 @@ def test_package_download_error(convert2rhel, shell, yum_cache):
     ) as c2r:
         c2r.expect("Validate the {} transaction".format(PKGMANAGER))
         c2r.expect("Adding {} packages to the {} transaction set.".format(SERVER_SUB, PKGMANAGER))
+
+        if "7" in SYSTEM_RELEASE_ENV:
```

Comment:
Just a heads up, this condition will also apply to `oracle-8.7` (and other clones when available).
I suppose it won't cause much of an issue, just to be careful with the vague conditioning.
Again, I guess it will be fine for the current latest, so no need to implement this, but prepending a dash `-` would be more specific.
```suggestion
        if "-7" in SYSTEM_RELEASE_ENV:
```
or even
```suggestion
        if re.match(r"^(centos|oracle)-7$", SYSTEM_RELEASE_ENV):
```
Other than that it looks good.
Thanks for the resolution!

Reply:
Thanks for the heads-up! I vaguely remember that we have some sort of function/class that returns a more constructed object with this information too, right?  I think it would be better to use it instead of regex.

Reply:
Indeed, there is the `system_release()` fixture, however I don't like it being dependent on the `/etc/os-release` or `/etc/system-release`
```
    Examples:
        Centos Linux 7.9 => centos-7.9\n
        Oracle Linux 8.6 => oracle-8.6\n
        Alma Linux 8.4 => almalinux-8.4\n
        Rocky Linux 8.5 => rocky-8.5
```

Reply:
Yeah and the `OsRelease` class, but the same story with `/etc/os-release`